config/production.yaml.example

```diff
@@ -53,6 +53,8 @@ redis:
 
 # SMTP server to send emails
 smtp:
+  transport: smtp
+  sendmail: null
```

Comment:
Add a comment `Path to sendmail command. Required if you use sendmail transport`